assets/css/admin/form-builder.css

```
@@ -7091,24 +7085,104 @@ button.swal2-cancel.swal2-styled.swal2-default-outline {
+  font-size: 1.125rem;
+  line-height: 1.75rem;
+  font-weight: 600;
+}
+
+.wpuf-ai-modal-close {
+  cursor: pointer;
+  border-width: 0px;
+  background-color: transparent;
+  font-size: 1.875rem;
+  line-height: 2.25rem;
+  line-height: 1;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

line-height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)